### DISCUSSION TITLE
`vim.lsp.handlers["textDocument/signatureHelp"]` has been overwritten

### DISCUSSION THREAD

**Discussion by TwelveNight** [LazyVim/LazyVim#941] [Q&A] [answered]
![QQ截图20230618141809](https://github.com/ray-x/lsp_signature.nvim/assets/113519679/54f6476e-bb5b-48ff-96e8-5a570c912168)

I am a neovim novice, how should I configure the error suggested in this message, I use LazyVim, I don’t know where to add the configuration

This is the plugin I add
```lua
-- hint when typing
return {
  {
    "ray-x/lsp_signature.nvim",
    event = "BufRead",
    config = function()
      require("lsp_signature").on_attach()
    end,
  },
}
```

**Comment by dpetka2001** [comment, marked as answer]
The error tells you what to do. In the same file just also put
```lua
  {
    "folke/noice.nvim",
    opts = {
      lsp = {
        signature = {
          enabled = false,
        },
      },
    },
  },
```
It should solve the error.

**Reply by TwelveNight** [reply]
Thank you for you help
